.github/workflows/end2end_cron.yml

```diff
@@ -1,82 +1,96 @@
 ---
-name: End-to-End Tests
+# yamllint disable
+
+name: 🚂 End-to-End Tests
 
 on:
-  push:
-    branches:
-      - master
-      - develop
-
-  pull_request:
-    branches:
-      - master
-      - develop
   schedule:
     - cron: 0 0 * * * # At Midnight Every Day
```

Comment:
I think it would make sense for something like the `develop` branch to have nightly builds, but we could probably accomplish that without adding to the CI. Implementing a `git pull develop` via a startup script would be pretty simple, and that image could be tagged(`twosixlabs/fortress:nightly`).

> Had we concluded that there was no gain to be had from nightly builds?
In short, no. Renamed and modified.



#2cool4review